integration-tests/gradle/update-dependencies.sh

```diff
@@ -37,8 +37,8 @@ echo ''
 set +o pipefail
 # note on sed: -deployment deps are added explicitly later and bom is upstream anyway
 ARTIFACT_IDS=$(grep -hR --include 'build*.gradle' --exclude-dir=build '[iI]mplementation' "${PRG_PATH}" | \
-              grep -Eo 'quarkus-[a-z0-9-]+' | \
-              sed -e '/-deployment/d' -e '/quarkus-bom/d' | sort | uniq)
+              grep -Eo ':quarkus-[a-z0-9-]+' | \
+              sed -e '/-deployment/d' -e '/quarkus-bom/d' -e 's/://' | sort | uniq)
```

Comment:
@famod could you please check this specific change?

Reply:
@famod, we reference dependencies that are build by an other project so they should be tracked by this script.

Reply:
Ok, I saw those `ext-quarkus-a` (b, c, etc.) test dependencies. I have not better suggestion given the required BSD compatibility.
Or maybe I have: _Not_ using `quarkus` in those deps is not an option?

Reply:
Yes we could use something Elsa that quarkus in the name. How this is solved in the maven plugin module?

Reply:
We could simply remove `quarkus` from the artifactIds, i.e. `ext-a`, etc.

Reply:
Yes I will revert this change and update project names